<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TimeDisplay</class>
 <widget class="QWidget" name="TimeDisplay">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>178</width>
    <height>39</height>
   </rect>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="DigitDisplay" name="m_hourDisplay" native="true">
     <property name="maximumSize">
      <size>
       <width>20</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>20</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="DigitDisplay" name="m_minDisplay" native="true">
     <property name="maximumSize">
      <size>
       <width>20</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>20</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="DigitDisplay" name="m_secDisplay" native="true">
     <property name="maximumSize">
      <size>
       <width>20</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>20</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>.</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="DigitDisplay" name="m_fracDisplay" native="true">
     <property name="maximumSize">
      <size>
       <width>20</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DigitDisplay</class>
   <extends>QWidget</extends>
   <header>ui/board/chrono/digitdisplay.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
